frontend/packages/ux-editor/src/components/Elements/Elements.tsx

```diff
@@ -75,13 +76,17 @@ export const Elements = () => {
             </div>
           </Accordion.Content>
         </Accordion.Item>
-        <Accordion.Item defaultOpen={true}>
-          <Accordion.Header>{t('left_menu.components')}</Accordion.Header>
-          <Accordion.Content>
-            {receiptLayoutName === selectedLayout ? <ConfPageToolbar /> : <DefaultToolbar />}
-          </Accordion.Content>
-        </Accordion.Item>
       </Accordion>
+      <div className={classes.componentsList}>
+        <div className={classes.componentsHeader}>
+          <Heading size='xxsmall'>
+            {t('left_menu.components')}
+          </Heading>
+          <Divider marginless />
+        </div>
```

Comment:
To spørsmål:
- Er det nødvendig å wrappe overskriften i en `div` her?
- Vi bør så langt det lar seg gjøre abstrahere stilrelatert kode fra dom-en. Har du prøvd å sette `border-bottom` på overskriften i stedet for å bruke `Divider`?

Reply:
Enig, endret nå 😁